fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: "Montserrat";
+  src: url("fonts/montserrat-regular-webfont.woff") format("woff");
+  src: url("fonts/montserrat-regular-webfont.woff2") format("woff2");
+  font-weight: normal;
+  font-style: normal;}
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6002411)